index.html

```diff
@@ -181,6 +181,31 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+  var wizardWidth = 70;
+  var wizardSpeed = 3;
+  var fireballSize = 22;
+
+  var getFireballSpeed = function(left) {
+    if (left = true) {
```

Comment:
здесь лучше использовать тернарный оператор:
```suggestion
    return left ? 5 : 2;
```